__tests__/apiTests.test.js

```diff
@@ -314,4 +315,34 @@ describe("GET /api/users",()=>{
                 })
             })
     })
+})
+describe("GET /api/articles?topicFilter",()=>{
+    test("get status code 200, and return all the articles with the specific topic filter",()=>{
+        return request(app)
+        .get("/api/articles?topic=mitch")
+        .expect(200)
+            .then(({body})=>{
+                const {articles} = body
+                expect(articles.length).toBeGreaterThan(0)
+                articles.forEach((article)=>{
+                    expect(article.topic).toBe("mitch")
+                })
+            })
+    })
+    test("get status code 400 if there are no topics with the topicFilter value",()=>{
+        return request(app)
+        .get("/api/articles?topic=paper")
```

Comment:
Should this be an error? This is a valid query for a topic that does exist in our database there just happens to be no articles associated with it. So really this should be a 200 empty response. So maybe we need a couple of tests for a 404 and a 200 here for topic does exist, or topic doesn't exist.